README.md

```diff
@@ -197,6 +195,9 @@ Removes a link from the node by name. Modifies the node.
 node.rmLink('Link1')
 ```
 
+#### `node.serialize()`
```

Comment:
I was skeptical about having a `node.serialize()` as I'm not a fan of having two ways of doing the same thing. Though I needed to do weird serializer import tricks due to circular dependencies which might not be needed anymore. Hence I'm supportive of this change. Therefore I think it also makes sense to add `node.deserialize()` for API symmetry in a subsequent PR.

Also `node.serialize()` can be seen as the DAP-PB API and the `util.*` calls are just the compatibility layer to make it work with js-ipld.